components/bl-rating-component/src/rating-icon.js

```diff
@@ -0,0 +1,69 @@
+import { icons } from './icons.js';
+
+const halfIconStyle = { width: `50%`, overflow: 'hidden', position: 'absolute' };
+
+export const RatingIcon = ({
+  icon,
+  iconColor,
+  index,
+  disabled,
+  ratingValue,
+  onChange
+}) => {
+  const halfInputValue = index + 0.5;
+  const inputValue = index + 1;
+
+  return (
+    <div className="bl-customComponent-ratingIcon rating-icon">
+      <label
```

Comment:
Could we split it to
`HalfPointControll` and `FullPointControll` components with `Label` and `Input` components inside?